core/src/main/java/com/netflix/conductor/metrics/Monitors.java

```diff
@@ -306,7 +306,19 @@ public static void recordAcquireLockFailure(String exceptionClassName) {
 		counter(classQualifier, "acquire_lock_failure", "exceptionType", exceptionClassName);
 	}
 
-	public static void recordSystemTaskWorkerPollingLimited(String queueName) {
+	public static void recordWorkflowCompleted(String workflowType) {
```

Comment:
We already have a Timer metric with `recordWorkflowCompletion`, which we traditionally convert to `counter` metric on the fly when needed. Can you provide more details about the need for a `counter` metric along with `timer` metric please.